### ISSUE TITLE
Learning Notebook - Part 3 of 3: incorrect retraining of encoder?

### ISSUE THREAD

**Issue by pma2** [LDSSA/batch2-BLU06#2]
In section "3.2 Meet the sklearn-like transformers"
The the fit+transform function is demostrated on the train set:
```
encoder = OrdinalEncoder(cols=['ComposerName'])
X_train_ = encoder.fit_transform(X_train)
X_train_.head()
```
The is said: "We can now use transform our test set."
And is used:
```
X_test_ = encoder.fit_transform(X_test)
X_test_.head()
```
But according to: http://contrib.scikit-learn.org/categorical-encoding/ordinal.html
**fit_transform: Fit to data, then transform it.**
Shouldn't be used instead:
**transform: Perform the transformation to new categorical data.**
To maintain the mapping consistency of the encoder between X_train_ and X_test_
?
Note: is done that way in the following cells (when defining the functions).
Thanks
